research-hub-web/src/app/app.component.html

```diff
@@ -1,301 +1,266 @@
-                      person
-                  </mat-icon>
-                  <span 
-                    fxHide 
-                    fxShow.gt-md>
-                      {{ userInfo?.firstName + ' ' + userInfo?.lastName }}
-                  </span>
-                  <span 
-                    fxHide 
-                    fxShow.md>
-                      {{ userInfo?.firstName }}
-                  </span>
-                  <span fxHide.gt-sm>
-                      {{ userInfo?.firstName?.charAt(0) + userInfo?.lastName?.charAt(0) }}
-                  </span>
-                </button>
-
-                <mat-menu 
-                  #userMenu="matMenu" 
-                  [overlapTrigger]="false">
-                  <div class="user-detail-dropdown">
-                    <img src="assets/images/uoa-logo-blue.svg">
-
-                    <mat-list class="user-menu-list">
-                      <mat-list-item>
-                        <mat-icon mat-list-icon>person</mat-icon>
-                        <p mat-line>{{ userInfo?.firstName + ' ' + userInfo?.lastName }}</p>
-                      </mat-list-item>
-                      <mat-list-item>
-                        <mat-icon mat-list-icon>note</mat-icon>
-                        <p mat-line>{{ userInfo?.email }}</p>
-                      </mat-list-item>
-                    </mat-list>
-
-                    <div 
-                      fxLayout="row" 
-                      fxLayoutAlign="end">
-                      <button 
-                        mat-raised-button 
-                        class="sign-out-button"
-                        href="https://www.auckland.ac.nz/en/about-us/about-the-university/identity-and-access-management/single-sign-on/details.html">
+
+                  <!-- User Details -->
+                  <mat-menu #userMenu="matMenu" [overlapTrigger]="false">
+                    <div class="user-detail-dropdown">
+                      <img src="assets/images/uoa-logo-blue.svg" alt="uoa-logo-blue">
+
+                      <mat-list class="user-menu-list">
+                        <mat-list-item>
+                          <mat-icon mat-list-icon>person</mat-icon>
+                          <p mat-line>{{ userInfo?.firstName + ' ' + userInfo?.lastName }}</p>
+                        </mat-list-item>
+                        <mat-list-item>
+                          <mat-icon mat-list-icon>note</mat-icon>
+                          <p mat-line>{{ userInfo?.email }}</p>
+                        </mat-list-item>
+                      </mat-list>
+
+                      <!-- Sign Out Button-->
+                      <div fxLayout="row" fxLayoutAlign="end">
+                        <button role="button" aria-label="sign-out" mat-raised-button class="sign-out-button"
+                          href="https://www.auckland.ac.nz/en/about-us/about-the-university/identity-and-access-management/single-sign-on/details.html">
                           Sign Out
```

Comment:
This button should actually log the user out - maybe add a logout function in app.component.ts e.g. 
logout() {
    this.loginService.logout();
}

Reply:
Logout working now :)